ENG4000-Team-A/capstone-project
# Add timestamp model

### New model has been created as a TimeStamp object.

### Specifications of the model
**hours -> Positive Integer Value (Can be up to any positive value)
minutes -> Integer Value within the boundary [0, 60] (Self explanatory)
seconds -> Integer Value within the boundary [0, 60] (Self explanatory)**

To be able to test changes
1. Open the repository directory and make sure to cd in to ../project/ConsoleTracker
2. Make sure to set up a virtual env to install dependences for the project (can install all using requirements.txt). Let me know if you have any trouble doing so
3. Create your personal superuser admin account using the command "python manage.py createsuperuser" 
4. Run "python manage.py runserver" to start web app on localhost
5. Visit the localhost URL + /admin to visit the admin site for our webapp
6. Log into your admin account
7. Try to create a new timestamp object with the constraints I gave above
